protocols/protocol/cloudstate/entity.proto

```diff
@@ -164,27 +272,69 @@ message Entity {
     string persistence_id = 3;
```

Comment:
https://github.com/cloudstateio/cloudstate/blame/master/docs/src/main/paradox/user/lang/java/eventsourced.md#L9 states that the persistence_id is optional:

```
It defaults to the simple name for the entity class (in this case, ShoppingCartEntity), 
it’s good practice to select one explicitly, this means your database isn’t depend 
on classnames in your code.
```

go-support has it implemented as optional but there are open questions if that is appropriate, see: https://github.com/cloudstateio/go-support/pull/5/files#r335443726